src/systems/filecoin_markets/storage_market/storage_deal/storage_deal_flow.md

```diff
@@ -30,17 +31,15 @@ Filecoin Storage Market Deal Flow
 
 - 7. Declared faults are penalized to a smaller degree than detected faults by `CronActor`. Miners declare failing sectors by invoking `StorageMinerActor.DeclareFaults` and X of the `StorageDealCollateral` will be slashed and power corresponding to these sectors will be tempororily lost. However, miners can only declare faults when they are not in `Challenged` status.
 - 8. Miners can then recover faults by invoking `StorageMinerActor.RecoverFaults` and have sufficient `StorageDealCollateral` in their available balances. FaultySectors are recommitted and power is only restored at the next PoSt submission. Miners will not be able to invoke `RecoverFaults` when they are in the `Challenged` status.
-- 9. Sectors that are failing for `storagemining.MAX_CONSECUTIVE_FAULTS` consecutive ChainEpochs will be cleared and result in `StoragePowerActor.SlashPledgeCollateral`.
+- 9. Sectors that are failing for `storagemining.MaxFaults` consecutive ChainEpochs will be cleared and result in `StoragePowerActor.SlashPledgeCollateral`.
```

Comment:
Agree that `MAX_CONSECUTIVE_FAULTS` is not the nicest of names, but if you want to rename it please do so everywhere consistently :D (right now it's defined in a ton of places)

Reply:
oops -- thought I fixed this one!